packages/core/README.md

```diff
@@ -1,4 +0,0 @@
-  TODO DOCS
```

Comment:
I realized this would look very similar to a converted version of core.adoc.

Personally I'd like to avoid duplicating content, but the package's readme is going to be visible in NPM. Should it maybe contain a link to the hosted documentation? Not sure.

Reply:
Yeah it would be more user friendly to have the info right on NPM, we could simply put some usage instructions and API reference in the Readme, maybe a generated js-doc.

Reply:
Let's leave that for now. Ideally readme should just point to documentation to not replicate content. I have the same issue on IOS. Cocoapods